### ISSUE TITLE
Error in rule makeKnownSpliceSites because of hista2_extract_splice_sites.py

### ISSUE THREAD

**Issue by Remi-Montagne** [maxplanck-ie/snakepipes#431]
Hello,

I am trying to use build indices for my organism by following the instructions of the README.md but I run into a problem when running the command

`createIndices --local --genomeURL /media/remi/Disk1/Yeast/data/ref_genomes/snakePipes_index/Scerevisiae_R64/Scerevisiae.fa --gtfURL /media/remi/Disk1/Yeast/data/ref_genomes/snakePipes_index/Scerevisiae_R64/Saccharomyces_cerevisiae.R64-1-1.96.gtf -o /home/remi/anaconda3/envs/snakePipes/lib/python3.6/site-packages/snakePipes/shared/organisms/ sacCer3`

Apparently snakePipes can't find #`hista2_extract_splice_sites.py`. Here the error message:
```
Failed to create temp dir under temp path prefix (/data/extended/)! Try fallback: /home/remi/anaconda3/envs/snakePipes/lib/python3.6/site-packages/snakePipes/shared/organisms/ ...
Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 5
Rules claiming more threads will be scaled down.
Job counts:
        count   jobs
        1       all
        1       makeKnownSpliceSites
        2

[Mon May 20 09:49:08 2019]
rule makeKnownSpliceSites:
    input: /home/remi/anaconda3/envs/snakePipes/lib/python3.6/site-packages/snakePipes/shared/organisms/annotation/genes.gtf
    output: /home/remi/anaconda3/envs/snakePipes/lib/python3.6/site-packages/snakePipes/shared/organisms/HISAT2Index/splice_sites.txt
    jobid: 10
    threads: 5

Activating conda environment: /home/remi/anaconda3/envs/ce6c00d970041b6854a508073a339ef7
/bin/bash: hista2_extract_splice_sites.py: command not found
[Mon May 20 09:49:09 2019]
Error in rule makeKnownSpliceSites:
    jobid: 10
    output: /home/remi/anaconda3/envs/snakePipes/lib/python3.6/site-packages/snakePipes/shared/organisms/HISAT2Index/splice_sites.txt
    conda-env: /home/remi/anaconda3/envs/ce6c00d970041b6854a508073a339ef7
    shell:
        
        hista2_extract_splice_sites.py /home/remi/anaconda3/envs/snakePipes/lib/python3.6/site-packages/snakePipes/shared/organisms/annotation/genes.gtf > /home/remi/anaconda3/envs/snakePipes/lib/python3.6/site-packages/snakePipes/shared/organisms/HISAT2Index/splice_sites.txt
        

Removing output files of failed job makeKnownSpliceSites since they might be corrupted:
/home/remi/anaconda3/envs/snakePipes/lib/python3.6/site-packages/snakePipes/shared/organisms/HISAT2Index/splice_sites.txt
Job failed, going on with independent jobs.
Exiting because a job execution failed. Look above for error message
Complete log: /home/remi/anaconda3/envs/snakePipes/lib/python3.6/site-packages/snakePipes/shared/organisms/.snakemake/log/2019-05-20T094907.051347.snakemake.log
Error: snakemake returned an error code of 1, so processing is incomplete!
```
Do you know what the problem is ?

Thanks and congratulation for the pipelines.

**Comment by dpryan79** [comment]
Hmm, which version do you have? I thought I had fixed this issue in the most recent one.

**Comment by dpryan79** [comment]
Oh, looks like there's a typo in there.

**Comment by Remi-Montagne** [comment]
I am using createIndices 1.2.0

**Comment by dpryan79** [comment]
I have this fixed in the develop branch now and will test it before pushing out version 1.2.1. That should happen today still.

**Comment by Remi-Montagne** [comment]
Great !

Thanks a lot. I am going to stay tune, then.

**Comment by dpryan79** [comment]
@Remi-Montagne This should be fixed in version 1.2.1, which is now available for install via conda. Thanks again for pointing this out and let me know if you run into other issues!

**Comment by Remi-Montagne** [comment]
Yes, it worked fine. I will be able to go to next step.

Thank you.
